tools/build_arrow.sh

```diff
@@ -3,75 +3,19 @@
 set -eu
 
 NPROC=$(nproc)
-
-TESTS=OFF
-BUILD_ARROW=OFF
```

Comment:
Hi @zhouyuan, was the `BUILD_ARROW=ON` option broken before the PR?

Reply:
no, will add back